src/Workspaces/Core/Portable/Workspace/Solution/Solution.cs

```diff
@@ -2056,18 +2050,28 @@ internal Task<bool> HasSuccessfullyLoadedAsync(Project project, CancellationToke
                 : project.Solution.GetProjectState(project.Id).HasAllInformation ? SpecializedTasks.True : SpecializedTasks.False;
         }
 
-        private static readonly ConditionalWeakTable<MetadataReference, ProjectId> s_metadataReferenceToProjectMap =
-            new ConditionalWeakTable<MetadataReference, ProjectId>();
+        /// <summary>
+        /// Symbols need to be either <see cref="IAssemblySymbol"/> or <see cref="IModuleSymbol"/>.
```

Comment:
It might be good to document what this map actually represents. This is more of a remark than a summary.